pinot-core/src/main/java/org/apache/pinot/core/segment/index/readers/ConstantValueDoubleDictionary.java

```diff
@@ -19,20 +19,25 @@
 package org.apache.pinot.core.segment.index.readers;
 
 /**
- * Dictionary for constant-value double
+ * Dictionary of a single double value.
  */
-
 public class ConstantValueDoubleDictionary extends BaseImmutableDictionary {
-  final Double _value;
+  private final double _value;
 
-  public ConstantValueDoubleDictionary(Double value) {
+  public ConstantValueDoubleDictionary(double value) {
     super(1);
     _value = value;
   }
 
   @Override
   public int insertionIndexOf(String stringValue) {
-
+    double doubleValue = Double.parseDouble(stringValue);
+    if (doubleValue < _value) {
+      return -1;
+    }
+    if (doubleValue > _value) {
+      return -2;
```

Comment:
I didnt knoiw we had this convention. The jacvadocs on insertionIndexOf() does not state this in comments. Also, it points to a binarysearch implementation that does not exist (or, I could not locate it).. Can you fix the comments to reflect this convention? thanks.

Reply:
Discussed offline, and that is the general binary search convention.